scripts/cluster_buildings.py

```diff
@@ -80,7 +80,9 @@ def get_central_points_geojson_with_buildings(
             }
         )
     central_features = gpd.GeoDataFrame(central_features, crs=microgrid_buildings.crs)
-    central_features.to_file(output_filepath_centroids)
+    if microgrid_buildings.crs != "EPSG:4326":
+        central_features = central_features.to_crs("EPSG:4326")
```

Comment:
You can add to line 82 the ".to_crs(...)" it doesn't harm

Reply:
Not sure to have understand what to do :D

Reply:
We can avoid the use of the if close, simply put the conversion to the crs when the file is opened

Reply:
Perfect, done!